recipes/stumpy/LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+STUMPY
```

Comment:
We can already include that in the `sdist` and would then not need to package the license here anymore: https://github.com/TDAmeritrade/stumpy/pull/4

Reply:
Does this mean that I will need to push a new version (1.1) to PyPI first? Or is it acceptable for now to leave the license out of this build?

Reply:
Yes, remove it when you update the package.